platforms/ios/src/TangramMap/TGTypes.mm

```diff
@@ -0,0 +1,11 @@
+//
+//  TGTypes.mm
+//  TangramMap
+//
+//  Created by Karim Naaji on 4/20/17.
+//  Copyright (c) 2017 Mapzen. All rights reserved.
+//
+
+#import "TGTypes.h"
+
+NSString* const TGErrorDomain = @"TGErrorDomain";
```

Comment:
I think that problem can be resolved by an extern version of this in the header, and then defining the string constant inside the implementation. Thoughts?